data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -384,8 +394,21 @@ def _prepare_initial_df(self) -> pd.DataFrame:
 
         census_tract_df = self._join_tract_dfs(census_tract_dfs)
 
-        # If GEOID10s are read as numbers instead of strings, the initial 0 is dropped,
-        # and then we get too many CBG rows (one for 012345 and one for 12345).
+        # Drop tracts that don't exist in the 2010 tracts
+        pre_join_len = census_tract_df.shape[0]
```

Comment:
an additional psychotic check -- I'm wondering if we should check for unique census tracts instead of just shape (e.g., `census_tract_df[GEOID10_TRACT].nunique()`)?

Reply:
Good thinking --- that is more precise and conveys better what we're trying to do. Will implement.